products/samsungmobile.md

```diff
@@ -0,0 +1,914 @@
+---
+layout: post
+permalink: /samsung
```

Comment:
we should use samsung-galaxy oder samsung-mobile.

"Samsung" could also be tablets, tvs etc.

Reply:
Thanks for your feedback, initially had it as Samsung Mobile but decided against it since there weren't any other Samsung entries.